doc/api/http.md

```diff
@@ -3206,6 +3206,50 @@ server.on('request', (request, res) => {
 server.listen(8000);
 ```
 
+## `http.createStaticServer([options])`
+
+<!--YAML
+added: REPLACEME
+-->
+
+> Stability: 0 - Experimental
+
+* `options` {Object}
+  * `directory` {string|URL} Root directory from which files would be served.
+    **Default:** `process.cwd()`.
+  * `port` {number}
+  * `host` {string} **Default:** `'localhost'`
+  * `mimeOverrides` {Object} Dictionary linking file extension to a MIME string,
+    to override or extend the built-in ones.
+  * `serveDotFiles` {boolean} **Default:** `false`
```

Comment:
Bikeshed: `s/serveDotFiles/allowDotFiles`

Just an alternative (non-blocking) suggestion, as an option to consider... define this as a custom `filter()` function where the default implementation filters out dot files... essentially something like:

```js
http.createStaticServer({
  directory: './files',
  port: 1234,
  filter(pathName) {
    // return true if the pathName is acceptable.
    // if filter is not provided, the default would block dot files or paths with dot segments (e.g. both `/.foo`, `/.foo/bar` would be rejected) 
  };
});
```